README.md

```diff
@@ -13,35 +13,86 @@ method. For example, using [kind](https://kind.sigs.k8s.io/):
 kind create cluster --name xnat
 ```
 
-### Package the chart
+### Install the CNPG Operator
+
+We use the [CNPG Operator](https://github.com/cloudnative-pg/cloudnative-pg) to
+deploy Postgres. The operator can be installed using Helm:
+
+```bash
+helm repo add cnpg https://cloudnative-pg.github.io/charts
+helm upgrade --install cnpg \
+  --namespace cnpg-system \
+  --create-namespace \
+  cnpg/cloudnative-pg
+```
+
+### Create a namespace to install the chart
+
+Create a manifest for the namespace:
+
+```bash
+cat <<EOF > namespace.yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: xnat-core
+EOF
+```
```

Comment:
The use of `--create-namespace` when running `helm install` means we don't necessarily need this. Maybe we should note that we would need this if `--create-namespace` was omitted?